pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
I'm terribly sorry, but I don't understand the point you're making.

AFAIK in ruby module and class are constants, and constants name must be unique.
for example you can see it online on repl.it : https://repl.it/Bxdv/0

I get the two conventions, even if I prefer the one with `VERSION` constant in the module/class

please could you help me understand by providing some lines in a gist.

regards,